instances/UBERONParcellation/ansiformLobuleCrusI.jsonld

```diff
@@ -4,34 +4,22 @@
     },
     "@id": "https://openminds.ebrains.eu/instances/UBERONParcellation/ansiformLobuleCrusI",
     "@type": "https://openminds.ebrains.eu/controlledTerms/UBERONParcellation",
-    "definition": "'Ansiform lobule crus I' is an ansiform lobule and hemisphere part of cerebellar posterior lobe.",
     "description": "",
```

Comment:
left over (I think this issue got lost in all my comments...): please set this to `"description": null,`

Reply:
Done.